### DISCUSSION TITLE
Unable to build new liquid library using wmake

### DISCUSSION THREAD

**Discussion by kyounes14** [blueCFD/Core#216] [Q&A] [answered]
While trying to add a new user-defined liquid in the src\thermophysicalModels\thermophysicalProperties\liquidProperties directory and re-making the library using wmake libso, I get the following error: 

![image](https://user-images.githubusercontent.com/36675456/45395132-99b89200-b602-11e8-8a7f-69f225d6bdde.png)

In this instance, my liquid is H2O2. 

Steps to reproduce the error: 
1- Create new folder in liquidProperties with .C, .H, and I.H files.
2- Update Make\files with path to new liquid 
3- Re-make library using libso
4- Error pops up while making dependencies

**Comment by wyldckat** [comment, marked as answer]
According to the message shown in the image, there is no problem, because the workaround was deployed without problems:

```
Error occurred with cv2pdb, have stripped binary as a workaround
```

The error "cannot load PDB helper DLL" occurs probably because you don't have installed Visual Studio installed. This is not a critical feature, it's just that it will not be able to get stack tracing on this library if it crashes when running one of its methods/functions.

**Comment by kyounes14** [comment]
I'd like to get stack tracing on it, if possible. So, I installed Visual Studio 2017, but the error persists. Is there any way I can avoid it? 

Edit: After installing VS 2017, adding Windows Symbols in the Debugger, and re-booting, the error disappears. However, the workaround message is still being called:

`(strip --strip-unneeded /home/ofuser/blueCFD/OpenFOAM-5.x/platforms/mingw_w64GccDPInt32Opt/lib/libthermophysicalProperties.dll; \
        /home/ofuser/blueCFD/OpenFOAM-5.x/wmake/scripts/infoCV2PDBworkaround)`

Does this imply successful execution?

**Comment by wyldckat** [comment]
If the message:

```
Error occurred with cv2pdb, have stripped binary as a workaround
```

is not shown, then it is not using the workaround.

And many thanks for the feedback! I'll add it to the FAQ when I get a chance.
